### ISSUE TITLE
[a11y]: TextArea [playground] - remaining character count not announced by SR

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#14818]
### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.32.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS & VO

### Description
As we type the input in the edit area the remaining character count is changing Dynamically but it is not announced by Screen Reader Instantly

If the screen reader fails to identify a dynamic content change, it affects users who are blind as they may not know about the change in the page.

### Recommended Fix
For the benefit of Screen Reader user its good to announce the Dynamic Changes instantly. Please refer the following link https://codepen.io/rikschennink/pen/LmoJYY

### WCAG 2.1 Violation

4.1.3 Status Messages

**Comment by guidari** [comment]
I'll close this PR for these reason:

- I just figured it out that the word count starts when there is 10 characters left to type (from 490 to 500 characters). The solution I inserted counts the characters from the beggining, like from 0 to 500 characters.
- We don't need to add third party code since the existing solution takes care of warning the user.
